backend/src/main/java/com/tourneynizer/tourneynizer/dao/RosterDao.java

```diff
@@ -18,13 +18,13 @@ public RosterDao(DataSource dataSource) {
         this.jdbcTemplate = new JdbcTemplate(dataSource);
     }
 
-    public void registerUser(User user, Team team, boolean isLeader) {
+    static void registerUser(User user, Team team, boolean isLeader, JdbcTemplate jdbcTemplate) {
```

Comment:
Why is this package-private?

Reply:
27999

Reply:
Why isn't a member function?

Reply:
It'd be tricky for other DAOs to have other DAOs due to the nature of Spring

Reply:
Ok then that makes sense